graylog2-web-interface/src/components/search/MessageTableEntry.jsx

```diff
@@ -1,10 +1,14 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import Immutable from 'immutable';
+import { Popover, OverlayTrigger } from 'react-bootstrap';
 
 import MessageDetail from './MessageDetail';
 import { Timestamp } from 'components/common';
 import StringUtils from 'util/StringUtils';
+import DateTime from 'logic/datetimes/DateTime';
+
+const style = require('!style!css!./MessageTableEntry.css');
```

Comment:
This produces an [`import/no-webpack-loader-syntax`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-webpack-loader-syntax.md) error in ESLint. You can replace it with a normal import:
`import style from './MessageTableEntry.css';`